src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball ball = new Ball();
```

Comment:
Here, you creating absolutely new and empty object of Ball class.
You have Lottery object, that has method to create a new ball.

instead of this, you have to `lottery.getRandomBall()` in for loop